### ISSUE TITLE
Review-2018.11.25-高寅超

### ISSUE THREAD

**Issue by yc-gao** [DIYer22/weekly-review#373]
# Review

- 阅读一篇分割相关的综述类(论文)[https://arxiv.org/abs/1704.06857]。
- 阅读了一下 MaskRCNN 论文。
- 了解了一下目标检测未来发展方向的情况，主要是搜知乎。

# Next
- 周二左右，将上周阅读信息进行整理归纳，主要形式笔记，放到申请的知乎专栏上面。
- 调试 MaskRCNN 代码。

**Comment by DIYer22** [comment]
```
(论文)[https://arxiv.org/abs/1704.06857] => [论文](https://arxiv.org/abs/1704.06857)
```
(论文)[https://arxiv.org/abs/1704.06857] => [论文](https://arxiv.org/abs/1704.06857)


MaskRCNN 代码打算调试哪一份啊?

**Comment by yc-gao** [comment]
> ```
> (论文)[https://arxiv.org/abs/1704.06857] => [论文](https://arxiv.org/abs/1704.06857)
> ```
> (论文)[https://arxiv.org/abs/1704.06857] => [论文](https://arxiv.org/abs/1704.06857)
> 
> MaskRCNN 代码打算调试哪一份啊?

我下载了三份代码，一份是论文中提到的[facebookresearch](https://github.com/facebookresearch/Detectron.git)，还有两份分别是[pytorch版](https://github.com/wannabeOG/Mask-RCNN.git)和[tensorflow版](https://github.com/matterport/Mask_RCNN.git)。我比较偏向于 pytorch 版的代码，相对简洁一点。

**Comment by DIYer22** [comment]
分享一些个人感受:      
我用过这个版本 [Detectron.pytorch](https://github.com/roytseng-tw/Detectron.pytorch), 但是坑很多 不推荐, 检测大牛同事推荐这个版本[maskrcnn-benchmark](https://github.com/facebookresearch/maskrcnn-benchmark)

**Comment by yc-gao** [comment]
> 分享一些个人感受:
> 我用过这个版本 [Detectron.pytorch](https://github.com/roytseng-tw/Detectron.pytorch), 但是坑很多 不推荐, 检测大牛同事推荐这个版本[maskrcnn-benchmark](https://github.com/facebookresearch/maskrcnn-benchmark)

学长，我在安装调试maskrcnn-benchmark 代码的时候，遇到了一个 bug。使用的是[docker](https://github.com/facebookresearch/maskrcnn-benchmark/tree/master/docker)环境。我搜索了一下这个 bug信息，[这里](https://github.com/docker/for-linux/issues/480) 有这个 bug 的记录，现在这个 bug 还处于开放状态。对于其中的赞最高的方法我尝试了一下：
```shell
echo N | sudo tee /sys/module/overlay/parameters/metacopy
```
报错，只读文件系统:

![2018-11-29 15-38-18](https://user-images.githubusercontent.com/36495505/49206365-d5d2e600-f3ec-11e8-85cc-86b9f58ecc00.png)

你有遇到这个问题么？

**Comment by DIYer22** [comment]
@xundaoxd 我这两周内才会切换到这个版本, 现在还在用旧版

**Comment by ilydouble** [comment]
旷世有个框架，改进了MASKRNN

**Comment by yc-gao** [comment]
> @xundaoxd 我这两周内才会切换到这个版本, 现在还在用旧版

我发现这个[仓库](https://github.com/matterport/Mask_RCNN.git) bug 比较少，不用怎么调整结果就 ok
